detectors/aws/ecs/ecs.go

```diff
@@ -17,9 +17,14 @@ package ecs // import "go.opentelemetry.io/contrib/detectors/aws/ecs"
 import (
 	"context"
 	"errors"
+	"fmt"
+	"net/http"
 	"os"
+	"regexp"
 	"strings"
 
+	ecsmetadata "github.com/brunoscheufler/aws-ecs-metadata-go"
```

Comment:
This repository has 3 stars and 14 commits. I wonder how reliable it can be in the future.
It's also fairly simple. So we can also extract its content and bring it back into this package later on if necessary.